css/style.css

```
@@ -0,0 +1,465 @@
+:root {
+    --basic-gray: #EEEEEE;
+    --basic-lighter-gray: #E1E1E1;
+    --basic-light-grayer: #B4B9BB;
+    --basic-greyer: #DFDFDF;
+    --basic-gray-border: #DBDBDB;
+    --basic-derker-gray: #D5D5D5;
+    --basic-black: #000000;
+    --basic-red: #FB565A;
+    --basic-dark-red:#E74246;
+    --basic-darker-red: #D7373B;
+    --basic-green:  #00CA74;
+    --basic-dark-green: #00BC6C;;
+    --basic-darker-green: #00AA62;
+    --basic-yellow: #EFC849;
+    --basic-dark-yellow: #EAB534;
+    --basic-darker-yellow: #E5A722;
+    --basic-white: #ffffff;
+    --basic-lightgray: #666666;
+    --basic-darkgray: #444444;
+    --basic-light-black: #283136;
+    --basic-black-ckeck-radio: #4D4D4D;
+
+    --special: #663d15;
+
+}
+
+body {
+  min-width: 960px;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--basic-black);
+  background-color: var(--basic-white);
+  background-position: top center;
+  background-repeat: no-repeat;
+}
+
+a {
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+.main-header-logo:hover {
+ opacity: 0.7;
+}
+.main-header-logo:active {
+ opacity: 0.3;
+
+}
+
```

Старайся аккуратнее писать, соблюдай отступы идентичными:

```suggestion

a {
  text-decoration: none;
}

img {
  max-width: 100%;
  height: auto;
}

.main-header-logo:hover {
  opacity: 0.7;
}

.main-header-logo:active {
  opacity: 0.3;
}

```